Integrations/CheckPhish/CheckPhish.py

```diff
@@ -0,0 +1,212 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+BASE_URL = demisto.params()['url']
+API_KEY = demisto.params().get('token')
+GOOD_DISP = demisto.params().get('good_disp')
+SUSP_DISP = demisto.params().get('susp_disp')
+BAD_DISP = demisto.params().get('bad_disp')
+USE_SSL = not demisto.params().get('insecure', False)
+handle_proxy('proxy')
+
+HEADERS = {
+    'Content-Type': 'application/json',
+    'Accept': 'application/json'
+}
+
+STATUS_SUFFIX = '/status'
+CLEAN_STATUS = 'clean'
+PENDING_STATUS = 'PENDING'
+DONE_STATUS = 'DONE'
+
+DEFAULT_GOOD_DISP = {'clean'}
+
+DEFAULT_SUSP_DISP = {'suspicious',
+                     'drug_spam'}
+
+DEFAULT_BAD_DISP = {'likely_phish',
+                    'phish'}
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, url, params=None, data=None):
+    res = requests.request(
+        method,
+        url,
+        verify=USE_SSL,
+        params=params,
+        data=data,
+        headers=HEADERS
+    )
+    if res.status_code not in {200}:
+        return_error('Error in API call to CheckPhish [%d] - %s' % (res.status_code, res.reason))
+
+    try:
+        return res.json()
+
+    except ValueError:
+        return None
+
+
+def unite_dispositions():
+    for disp in GOOD_DISP:
+        DEFAULT_GOOD_DISP.add(disp)
+
+    for disp in SUSP_DISP:
+        DEFAULT_SUSP_DISP.add(disp)
+
+    for disp in BAD_DISP:
+        DEFAULT_BAD_DISP.add(disp)
+
+
+def get_dbot_score(disposition):
+    if disposition in DEFAULT_BAD_DISP:
+        return 3
+    if disposition in DEFAULT_SUSP_DISP:
+        return 2
+    if disposition in DEFAULT_GOOD_DISP:
```

Comment:
Not sure the Good should be 1, as it is marked as `Low` @mayagoldb what do you think?